tools/RAiDER/checkArgs.py

```diff
@@ -31,6 +31,7 @@ def checkArgs(args):
 
     os.makedirs(args.output_directory, exist_ok=True)
     os.makedirs(args.weather_model_directory, exist_ok=True)
+    args['weather_model'].set_wmLoc(args.weather_model_directory)
```

Comment:
the directory with the weather model files is now an attribute of the class that can be fetched at any time